frontend/src/app/shared/components/okr-toolbar/okr-toolbar.component.ts

```diff
@@ -78,4 +79,9 @@ export class OkrToolbarComponent implements OnInit {
       .subscribe();
   }
 
+  pickLanguageClicked() {
+    this.dialog.open(PickLanguageComponent)
+      .afterClosed()
+      .subscribe();
```

Comment:
Subscription ohne unsubscribe

Reply:
Jo ,ist bei den Dialogen in der ganzen App so und das ist auch richtig, weil das Observable (Dialog) nach dem Schließen "finalized/completed" ist und keine neuen Werte ausgibt.